testbed/testbed/senders.go

```diff
@@ -73,13 +73,17 @@ type MetricDataSenderOld interface {
 // DataSenderOverTraceExporter partially implements TraceDataSender via a TraceExporter.
 type DataSenderOverTraceExporterOld struct {
 	exporter component.TraceExporterOld
+	Host     string
```

Comment:
Do we ever use anything other than locahost in the tests? I cannot find code that does so.

Reply:
Will be different if running the otelcol(s) under test via k8s or docker-compose